app/scripts/modules/amazon/src/loadBalancer/configure/classic/AdvancedSettings.tsx

```diff
@@ -1,17 +1,17 @@
 import * as React from 'react';
-import { Field, FormikProps } from 'formik';
+import { Field, FormikErrors } from 'formik';
 
 import { HelpField, IWizardPageProps, ValidationMessage, wizardPage } from '@spinnaker/core';
 
 import { IAmazonClassicLoadBalancerUpsertCommand } from 'amazon/domain';
 
-class AdvancedSettingsImpl extends React.Component<
-  IWizardPageProps & FormikProps<IAmazonClassicLoadBalancerUpsertCommand>
-> {
+export type IAdvancedSettingsProps = IWizardPageProps<IAmazonClassicLoadBalancerUpsertCommand>;
```

Comment:
Would like feedback here.

this
```ts
export type IAdvancedSettingsProps = IWizardPageProps<IAmazonClassicLoadBalancerUpsertCommand>;
```

can also be written

```ts
export interface IAdvancedSettingsProps extends IWizardPageProps<IAmazonClassicLoadBalancerUpsertCommand> {
};
```

and in other places we're actually extending IWizardPageProps<T>, such as:

```ts
export interface ITargetGroupsProps extends IWizardPageProps<IAmazonApplicationLoadBalancerUpsertCommand> {
  app: Application;	  app: Application;
  isNew: boolean;	  isNew: boolean;
  loadBalancer: IAmazonApplicationLoadBalancer;	  loadBalancer: IAmazonApplicationLoadBalancer;
@@ -26,20 +26,17 @@ export interface ITargetGroupsState {
  oldTargetGroupCount: number;	  oldTargetGroupCount: number;
}	}
```

I prefer the `type` to the empty `IFoo interface extends IBar { }` with an empty block.  However, since we're extending elsewhere, there's an argument to be made to "be consistent and always extend the interface"

Reply:
I like `type`.